Gemfile

```diff
@@ -16,16 +16,18 @@ gem 'mongoid'
 gem 'mongoid_paranoia'
 gem 'oai'
 gem 'puma'
-gem 'rails', '~> 6.1.4.6'
+gem 'rails', '~> 7.0.2'
 gem 'redis', '~> 4.0' # for action_cable in production
 gem 'render_async', '~> 2.1', '>= 2.1.11'
 gem 'rqrcode'
 gem 'server_timing'
-gem 'supplejack_common', github: 'DigitalNZ/supplejack_common', tag: 'v2.10.6.1'
+gem 'supplejack_common', github: 'DigitalNZ/supplejack_common', branch: 'pm/rails-7'
```

Comment:
Can we merge this branch and make a version a now all supplejack components are in Rails 7?

Reply:
What do you mean sorry? Why do you want to merge that now?

Reply:
I created the tag and used it here :)